packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -62,14 +105,47 @@ const MainContent = ( {
 		: __(
 				'You have not yet created any menus. Displaying a list of your Pages'
 		  );
+
 	return (
-		<OffCanvasEditor
-			blocks={ clientIdsTree }
-			parentClientId={ clientId }
-			isExpanded={ true }
-			LeafMoreMenu={ LeafMoreMenu }
-			description={ description }
-		/>
+		<div className="wp-block-navigation__menu-inspector-controls">
```

Comment:
Can we avoid this extra div?

Reply:
yes, we need the classname on a div, not the table element for the horizontal scroll to work